implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -305,6 +313,8 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
             contextInfo.setTokenHeader(tokenHeader);
         }
 
+        contextInfo.setAlwaysCheckAutorization(alwaysCheckAuthorization);
+
```

Comment:
This class also has a copy constructor and the property getters in the end, please update those

Reply:
I updated the property getters, but what do you mean by copy constructor.

Reply:
@nicolaimainiero sorry, [meant this factory method](https://github.com/smallrye/smallrye-jwt/blob/master/implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java#L73).